src/components/Text/Text.styles.ts

```diff
@@ -1,139 +1,121 @@
     ${baseStyles};
 
-    font-size: ${oldTypography.sizes.body1};
-    line-height: ${oldTypography.lineHeights.body1};
-    font-weight: ${oldTypography.weights.regular};
+    font: ${cVar('typographyDesktopT300')};
+    letter-spacing: ${cVar('typographyDesktopT300LetterSpacing')};
+    text-transform: ${cVar('typographyDesktopT300TextTransform')};
   `,
-  body2: styled.p<TextProps>`
+  't300-strong': styled.span<TextProps>`
     ${baseStyles};
 
-    font-size: ${oldTypography.sizes.body2};
-    line-height: ${oldTypography.lineHeights.body2};
-    font-weight: ${oldTypography.weights.regular};
+    font: ${cVar('typographyDesktopT300Strong')};
+    letter-spacing: ${cVar('typographyDesktopT300StrongLetterSpacing')};
+    text-transform: ${cVar('typographyDesktopT300StrongTextTransform')};
   `,
-  caption: styled.span<TextProps>`
+  t200: styled.p<TextProps>`
     ${baseStyles};
 
-    font-size: ${oldTypography.sizes.caption};
-    line-height: ${oldTypography.lineHeights.caption};
-    font-weight: ${oldTypography.weights.regular};
+    font: ${cVar('typographyDesktopT200')};
+    letter-spacing: ${cVar('typographyDesktopT200LetterSpacing')};
+    text-transform: ${cVar('typographyDesktopT200TextTransform')};
   `,
-  overhead: styled.span<TextProps>`
+  't200-strong': styled.span<TextProps>`
     ${baseStyles};
 
-    font-size: ${oldTypography.sizes.overhead};
-    line-height: ${oldTypography.lineHeights.overhead};
-    font-weight: ${oldTypography.weights.semibold};
+    font: ${cVar('typographyDesktopT200Strong')};
+    letter-spacing: ${cVar('typographyDesktopT200StrongLetterSpacing')};
+    text-transform: ${cVar('typographyDesktopT200StrongTextTransform')};
   `,
-  button1: styled.span<TextProps>`
+  t100: styled.span<TextProps>`
     ${baseStyles};
 
-    font-size: ${oldTypography.sizes.button.large};
-    line-height: ${oldTypography.lineHeights.button};
-    font-weight: ${oldTypography.weights.bold};
-    font-family: ${oldTypography.fonts.headers};
+    font: ${cVar('typographyDesktopT100')};
+    letter-spacing: ${cVar('typographyDesktopT100LetterSpacing')};
+    text-transform: ${cVar('typographyDesktopT100TextTransform')};
   `,
-  button2: styled.span<TextProps>`
+  't100-strong': styled.span<TextProps>`
     ${baseStyles};
 
-    font-size: ${oldTypography.sizes.button.medium};
-    line-height: ${oldTypography.lineHeights.button};
-    font-weight: ${oldTypography.weights.bold};
-    font-family: ${oldTypography.fonts.headers};
-  `,
-  button3: styled.span<TextProps>`
-    ${baseStyles};
-
-    font-size: ${oldTypography.sizes.button.small};
-    line-height: ${oldTypography.lineHeights.button};
-    font-weight: ${oldTypography.weights.bold};
-    font-family: ${oldTypography.fonts.headers};
+    font: ${cVar('typographyDesktopT100Strong')};
+    letter-spacing: ${cVar('typographyDesktopT100StrongLetterSpacing')};
+    text-transform: ${cVar('typographyDesktopT100StrongTextTransform')};
```

Comment:
Any specific reason to make `t300` and `t200` render as `p` but `t300-strong`, `t200-strong`, `t100` and `t100-strong` as `span`?

Reply:
no reason really it was all there already and then I reordered them, should we make it all `p`'s?

Reply:
Maybe let's try `span`, we don't really have that much content that should go into paragraphs